VideoLocker/res/values/errors.xml

```diff
@@ -63,6 +63,19 @@
     <!-- Alert dialog content shown when user attempts to view a video that hasn't been synced -->
     <string name="video_offline_message">Please select a video that you’ve downloaded onto your device.</string>
 
+    <!-- New version available -->
+    <!-- Snackbar notification shown when the current version of the app is unsupported -->
+    <string name="app_version_unsupported">Your version of the app is no longer supported. Upgrade to the latest version.</string>
```

Comment:
This change has already been done in commit eb6b4d8eb6ea93c9f5a4ea6d62f2f85cad589abb. You may have been looking at an older commit.